src/app/pages/registration/register-form/register-form.component.html

```diff
@@ -0,0 +1,150 @@
+<div class="container">
+  <div class="details">
+    <div class="information">
+      <div class="left-section">
+        <h4>Create a free account today!</h4>
+        <form [formGroup]="registerForm" (ngSubmit)="onSubmit(registerForm.value)">
+          <div class="input-wrap">
+              <div class="input-icons"> 
+                  <i class="fa fa-envelope icon"> 
+                </i> 
+            <input 
+            formControlName="email"
+            type="text" class="email"
+              placeholder="Email address" />
+              <div *ngIf="registerForm.controls['email'].invalid && registerForm.controls['email'].touched && registerForm.controls['email'].dirty" class="alert alert-danger">
```

Comment:
great work @akram256, I realize that when a user touches and then leaves the email formControl, a blank error is displayed maybe we can do better by adding an email is required error in there.
<img width="515" alt="Screenshot 2019-07-24 at 15 42 13" src="https://user-images.githubusercontent.com/20795487/61794764-871a0680-ae2a-11e9-81df-8eb35f978df4.png">

Reply:
Hey I worked on this.